src/transformers/models/vipllava/modeling_vipllava.py

```diff
@@ -418,26 +418,48 @@ def forward(
             vision_feature_layers if vision_feature_layers is not None else self.config.vision_feature_layers
         )
 
+        if (input_ids is None) ^ (inputs_embeds is not None):
+            raise ValueError(
+                "You cannot specify both input_ids and inputs_embeds at the same time, and must specify either one"
+            )
+
+        if pixel_values is not None and inputs_embeds is not None:
+            raise ValueError(
+                "You cannot specify both pixel_values and inputs_embeds at the same time, and must specify either one"
+            )
```

Comment:
Both can be `None`, that happens when we are in decoding stage and get only the last token for generation. We only can't have both set to a tensor, because that means we don't know where are `image_token` so we can't merge images into embeddings